modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/RequiredMark.tsx

```diff
@@ -8,7 +8,7 @@ import React from 'react';
 
 const RequiredMark = () => (
```

Comment:
It seems a good candidate to be moved to a shared place as it appears in 3 different modules with the same markup :wink: (adaptive-media, frontend-data-set & frontend-data-set-view)

Reply:
See https://github.com/liferay/clay/issues/5083

Reply:
Great